hooks/cache_location.py

```diff
@@ -12,6 +12,7 @@
 Hook to control path cache and bundle cache folder creation.
 """
 
+from __future__ import absolute_import
```

Comment:
I would argue that `from __future__ import absolute_import` is probably not necessary in our case. That would reduce noise in this PR if we removed it.

Reply:
👍 I removed many of them but my grep missed the hookds dir.